### PR TITLE
Schema changes 138, 137, 141, 143, 159, 160, 165, 161, 163

### PR DESCRIPTION
**Pull request by EasyRhin0**

### CODE REVIEW

File: src/Sarif/Schemata/Sarif.schema.json [hunk 1]
```diff
@@ -859,7 +900,7 @@
         },
 
         "kind": {
-          "description": "The type of construct this logicalLocationComponent refers to. Should be one of 'function', 'member', 'module', 'namespace', 'package', 'resource', or 'type', if any of those accurately describe the construct.",
+          "description": "The type of construct this logicalLocationComponent refers to. Should be one of 'function', 'variable', 'member', 'module', 'namespace', 'package', 'parameter', 'resource', 'returnType', or 'type', if any of those accurately describe the construct.",
```
Comment (lgolding) [inline src/Sarif/Schemata/Sarif.schema.json]: Let's alphabetize these. I'll fix the copy in the OASIS repo.

Comment (EasyRhin0) [reply src/Sarif/Schemata/Sarif.schema.json]: OK

**Comment by lgolding** [comment]
I'm sorry, you already did #145 and #155. We good.

**Comment by lgolding** [comment]
We are now missing _only_ the "fix" portion of #97.

**Comment by EasyRhin0** [comment]
Yep already working on 97.

**Pull request merged**